frontend/src/i18n/locales/en.json

```diff
@@ -0,0 +1,43 @@
+{
+  "Home": {
+    "header": "Send Stealth Payments",
+    "description": "Only the sender and recipient know who received funds",
+    "Tutorial": {
+      "tutorial": "Tutorial",
+      "receiving-funds": "Receiving Funds",
+      "receive-first": "Use the <span class=\"text-bold\">Setup</span> page to configure your account",
+      "receive-second": "Ask someone to send funds to your address, ENS, or CNS name through the Umbra app",
+      "receive-third": "Check the <span class=\"text-bold\">Receive</span> page to withdraw funds",
+      "sending-funds": "Sending Funds",
+      "send-first": "Get the address, ENS, or CNS name of the person you're sending funds to",
+      "send-second": "Complete the form on the <span class=\"text-bold\">Send</span> page"
+    }
+  },
+  "Home-action-card": {
+    "send": "Send",
+    "send-description": "Send funds to another user",
+    "receive": "Receive",
+    "receive-description": "View and withdraw received funds",
+    "setup": "Setup",
+    "setup-description": "Setup your account to receive funds"
+  },
+  "Header-Links": {
+    "home": "Home",
+    "FAQ": "FAQ",
+    "contact": "Contact"
+  },
+  "Base-Layout": {
+    "settings": "Settings",
+    "dark-mode": "Dark mode",
+    "advanced-mode": "Advanced mode",
```

Comment:
I noticed the tooltip that is shown when hovering over the advanced mode indicator in the header is not translated.

Also, just out of curiosity, do you have any idea what other projects usually do about network names? Are there "translations" for these or are they typically left in English?

<img width="477" alt="Screen Shot 2022-04-19 at 11 24 54 AM" src="https://user-images.githubusercontent.com/1479802/164043247-e8dcaa99-7192-47ec-b9fc-8e76923d5369.png">